Java-Majasdarbi/src/main/java/advanced/test/Exercise4/FileReaderAndSaver.java

```diff
@@ -0,0 +1,53 @@
+package advanced.test.Exercise4;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+public class FileReaderAndSaver {
+    public static void main(String[] args) throws IOException {
+        String filePath = "src/main/java/advanced/test/Exercise4/someText.txt";
+        String reversedString = getString(filePath);
+        writeFile(reversedString);
+    }
+
+    public static void writeFile(String str) {
+        try {
+            FileWriter fileWriter = new FileWriter("src/main/java/advanced/test/Exercise4/newText.txt");
+            fileWriter.write(str);
+            fileWriter.close();
+        } catch (IOException e) {
+            e.getStackTrace();
```

Comment:
Kāpēc šis ir šeit?

Reply:
No uzdevuma sapratu ka vajag izvedot jaunu failu ar to tekstu iekšā tajā pašā folderī!

Reply:
Ir runa par konkrēto rindiņu :)
`e.getStackTrace();`

Reply:
Sākumā liku to e.getMessage(),bet man radija pasvītrotu, tāpēc paskatījos kas ir līdzīgs viņam. 
To tākā nevajadzētu izmantot?